src/main/game/CommandLine/CommandlineUi.java

```diff
@@ -103,6 +122,23 @@ public void askForLanguage() {
         output.println(english.askForLanguage());
     }
 
+    private String getTranslatedMove(String userMove) {
+        return translations.get(userMove);
+    }
+
+    private boolean languageIsEnglish() {
+        return language instanceof English;
+    }
+
+    private Moves getConvertedMove(String userMove) {
+        if (languageIsEnglish()) {
+            return convertMove(userMove);
+        } else {
+            String translatedMove = getTranslatedMove(userMove);
+            return convertMove(translatedMove);
+        }
+    }
+
     private void translateToEnglish() {
```

Comment:
How come we need this Map if we are passing the language in through the constructor? Is this a hint at that the `Language` class (with English and Greek) does not yet fully model what "language" means our application?